stable/concourse/README.md

```diff
@@ -316,19 +301,18 @@ By default, this chart will not use a [Credential Manager](https://concourse.ci/
 ## ref: https://concourse.ci/creds.html
 ##
 credentialManager:
-  ## Enable Credential Manager using below configuration.
-  ##
-  enabled: true
 
-  ## use Hashicorp Vault for Credential Manager.
-  ##
   vault:
+    ## Use Hashicorp Vault for the Credential Manager.
+    ##
+    enabled: false
+
     ## URL pointing to vault addr (i.e. http://vault:8200).
     ##
-    url: http://vault:8200
+    # url:
 
-    ## initial periodic token issued for concourse
-    ## ref: https://www.vaultproject.io/docs/concepts/tokens.html#periodic-tokens
```

Comment:
I think it might be best to leave this comment as it specifies the kind of token needed. For example if you configured it with a non renewable token it would fail after a period of time and if you use (gasp) a root token it would be a security horror.